src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -319,7 +319,7 @@
                                             {% endif %}
                                         </td>
                                         <td>
-                                            {{ call.duration }}ms
+                                            {{ '%0.2f ms'|format(call.duration * 1000) }}
```

Comment:
`call.duration` unit already is milliseconds so it shouldn’t be multiplied by 1000 (security panel gets microseconds).

Reply:
Ah yes, true, thanks for pointing this out!